README.md

```diff
@@ -16,66 +16,30 @@ This repo is versioned with [Lerna](https://github.com/lerna/lerna).
 Follow these steps for setup:
 
 1. Pull the repo `git clone https://github.com/NYPL/nypl-design-system.git`
-2. Install Lerna globally `npm i -g lerna`
-3. Install all the packages `npm run install-all`
-4. Run `npm start`.
+2. Install the module `npm install`
```

Comment:
This file describes this PR pretty neatly 👏🏻🎉

Reply:
🎉 🎉